### ISSUE TITLE
Incorrect value for 10GHz frequency

### ISSUE THREAD

**Issue by sq6emm** [ok2cqr/cqrlog#442]
Hi,

on cqrlog 2.5.2 when you log QSO on 10GHz band you will see frequency like:

1,0250.000 instead of 10.250.000 or 10250.000 - you can see this on QSO list and when entering QSO.

For 1.3GHz band it is logged like: 1296.000 so either "." should be added or not but for sure 10GHz is incorrect.

vy 73 de SQ6EMM

**Comment by phl0** [comment]
Having a quick look at the sources I couldn't identify the error. But I have an idea. Maybe the frequencies > 10GHz are not formatted properly. 
Could you please check if this also applies to the default settings for 24, 47 and 78 Ghz pse?

**Comment by sq6emm** [comment]
Hi,

yes it looks like formatting problem, the same on all high bands.

**Comment by OH1KH** [comment]
Could it be how system locale is set to format numbers?

**Comment by sq6emm** [comment]
Good question..
I am using Ubuntu 20.04 LTS (with all updates)
Below locale:

```
LANG=en_US.UTF-8
LANGUAGE=
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=pl_PL.UTF-8
LC_TIME=pl_PL.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=pl_PL.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=pl_PL.UTF-8
LC_NAME=pl_PL.UTF-8
LC_ADDRESS=pl_PL.UTF-8
LC_TELEPHONE=pl_PL.UTF-8
LC_MEASUREMENT=pl_PL.UTF-8
LC_IDENTIFICATION=pl_PL.UTF-8
LC_ALL=
```

For test I've run:

`LC_ALL=en_US.utf8 cqrlog`

Issue remains the same.

**Comment by sq6emm** [comment]
Line 146 of https://github.com/ok2cqr/cqrlog/blob/master/src/fFreq.pas

Looks suspicious to me.

**Comment by sq6emm** [comment]
Fixed in: https://github.com/ok2cqr/cqrlog/pull/443

**Comment by sq6emm** [comment]
Done... Please review;)

**Comment by phl0** [comment]
I can confirm that this fix works.:

![Screenshot from 2021-09-30 14-31-30](https://user-images.githubusercontent.com/7112907/135455540-738d146e-a7c0-4785-9c83-039c35894887.png)

All GHz Band get logged correctly now. Thank you for fixing this. Guess I need to build some equipment for those bands now :-)

**Comment by sq6emm** [comment]
Yea... would be great to test hamlib + LO shift with this;)

**Comment by phl0** [comment]
tbh I don't know how that works. The bands table allows for configuration of RX/TX offsets. But I guess I need to read the manual about how these work.

**Comment by sq6emm** [comment]
Changes, merged in #443
